ports/nrf/boards/wt51822-s4at/mpconfigboard.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016 Glenn Ruben Bakke
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+#define WT51822_S4AT
+
+// Datasheet for board:
+// https://4tronix.co.uk/picobot2/WT51822-S4AT.pdf
+#define MICROPY_HW_BOARD_NAME       "WT51822-S4AT"
+#define MICROPY_HW_MCU_NAME         "NRF51822"
+#define MICROPY_PY_SYS_PLATFORM     "nrf51"
+
+#define MICROPY_PY_MACHINE_HW_SPI   (0)
```

Comment:
I suggest that we enable this one and adding the defines to the bottom of the pins used for SPI. It's hard to tell the usecase for everyone. If the SPI is not used, the pins are not occupied by any peripheral hardware.

config could look like this:
https://github.com/tralamazza/micropython/blob/master/ports/nrf/boards/microbit/mpconfigboard.h#L61-L65

When it comes to which pins to use its not many to choose from, but i suggest that staying away from P0.04 is a good guideline, which is the only routed analog pin (AIN5).